client/devdocs/design/component-examples.js

```diff
@@ -1,93 +1,94 @@
-export JetpackColophonExample from 'components/jetpack-colophon/docs/example';
-export JetpackHeaderExample from 'components/jetpack-header/docs/example';
-export JetpackLogoExample from 'components/jetpack-logo/docs/example';
-export LanguagePicker from 'components/language-picker/docs/example';
-export LineChart from 'components/line-chart/docs/example';
-export ListEnd from 'components/list-end/docs/example';
-export MarkedLinesExample from 'components/marked-lines/docs/example';
-export MultipleChoiceQuestionExample from 'components/multiple-choice-question/docs/example';
-export Notices from 'components/notice/docs/example';
-export PaginationExample from 'components/pagination/docs/example';
-export PaymentLogo from 'components/payment-logo/docs/example';
-export PieChart from 'components/pie-chart/docs/example';
-export Popovers from 'components/popover/docs/example';
-export ProgressBar from 'components/progress-bar/docs/example';
-export Ranges from 'components/forms/range/docs/example';
-export Rating from 'components/rating/docs/example';
-export Ribbon from 'components/ribbon/docs/example';
-export ScreenReaderTextExample from 'components/screen-reader-text/docs/example';
-export SearchDemo from 'components/search/docs/example';
-export SectionHeader from 'components/section-header/docs/example';
-export SectionNav from 'components/section-nav/docs/example';
-export SegmentedControl from 'components/segmented-control/docs/example';
-export SelectDropdown from 'components/select-dropdown/docs/example';
-export ShareButton from 'components/share-button/docs/example';
-export SiteTitleControl from 'components/site-title/docs/example';
-export SocialLogos from 'social-logos/example';
-export Spinner from 'components/spinner/docs/example';
-export SpinnerButton from 'components/spinner-button/docs/example';
-export SpinnerLine from 'components/spinner-line/docs/example';
-export SplitButton from 'components/split-button/docs/example';
-export Suggestions from 'components/suggestions/docs/example';
-export TextareaAutosize from 'components/textarea-autosize/docs/example';
-export TextDiff from 'components/text-diff/docs/example';
-export TileGrid from 'components/tile-grid/docs/example';
-export TimeSince from 'components/time-since/docs/example';
-export Timezone from 'components/timezone/docs/example';
-export TokenFields from 'components/token-field/docs/example';
-export Tooltip from 'components/tooltip/docs/example';
-export UserItem from 'components/user/docs/example';
-export Version from 'components/version/docs/example';
-export VerticalMenu from 'components/vertical-menu/docs/example';
-export VerticalNav from 'components/vertical-nav/docs/example';
-export Wizard from 'components/wizard/docs/example';
-export WizardProgressBar from 'components/wizard-progress-bar/docs/example';
-export WpcomColophon from 'components/wpcom-colophon/docs/example';
+export { default as ActionCard } from 'components/action-card/docs/example';
```

Comment:
Note: these changes were made to appease ESLint, which doesn't handle `eslint-disable` correctly for parsing errors, such as the one the previous syntax caused.

Reply:
Outdated.